### PR TITLE
Add support to enable cerberus

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds support to install and setup cerberus as a container
on the orchestration host where the OCP cluster is installed and
orchestrated.

Fixes https://github.com/openshift-scale/scale-ci-pipeline/issues/78

### CODE REVIEW

File: properties-files/openshiftv4_on_aws.properties [hunk 1]
```diff
@@ -12,6 +12,23 @@ SCALE_CI_BUILD_TRIGGER_URL=
 OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=
 OPENSHIFT_INSTALL_BINARY_URL=
 ENABLE_DITTYBOPPER=true
+CERBERUS_ENABLE=true
+KUBECONFIG_PATH=/root/.kube/config
+CERBERUS_CONFIG_PATH=/root/cerberus.yml
+CERBERUS_IMAGE=quay.io/openshift-scale/cerberus:latest
+CERBERUS_URL=http://0.0.0.0:8080
+WATCH_NODES=true
+WATCH_NAMESPACES=[openshift-etcd, openshift-apiserver, openshift-kube-apiserver, openshift-monitoring, openshift-kube-controller, openshift-machine-api, openshift-kube-scheduler, openshift-ingress, openshift-sdn, openshift-ovn-kubernetes]
```
Comment (mohit-sheth) [inline properties-files/openshiftv4_on_aws.properties:21]: "openshift-sdn" will take care of monitoring either openshift-sdn ns or openshift-ovn-kubernetes ns

File: jjb/dynamic/scale-ci_install_aws.yml [hunk 2]
```diff
@@ -110,6 +110,74 @@
...
+    - string:
+        default: ""
+        description: slack alias to ping when cop id's are not defined
+        name: SLACK_TEAM_ALIAS
+    - string:
+        default: "5"
+        description: number of iterations to run when daemon mode is not enabled
+        name: ITERATIONS
+    - string:
+        default: "60"
```
Comment (dry923) [inline jjb/dynamic/scale-ci_install_aws.yml:174]: Do we want to think about reducing this time at all? Maybe down to 30 seconds?

**Comment by mohit-sheth** [comment]
Do we want to add it Azure and GCP too?

File: properties-files/openshiftv4_on_aws.properties (hunk 1 above)
Comment (chaitanyaenr) [reply properties-files/openshiftv4_on_aws.properties]: Fixed it, thanks.

File: jjb/dynamic/scale-ci_install_aws.yml (hunk 2 above)
Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci_install_aws.yml]: 30 seconds default makes sense, we can tweak the value in the properties file depending on the use case. Fixed it, thanks.

**Comment by chaitanyaenr** [comment]
@mohit-sheth @dry923 Also added support to enable cerberus in Azure and GCP jobs in addition to AWS like suggested. PTAL. Thanks.

**Comment by chaitanyaenr** [comment]
Depends on https://github.com/openshift-scale/scale-ci-deploy/pull/77.

**Pull request merged**